packages/core/src/modules/proofs/formats/models/ProofFormatServiceOptions.ts

```diff
@@ -35,7 +35,7 @@ export interface ProcessRequestOptions {
   record?: ProofExchangeRecord
 }
 
-export interface FormatCreatePresentationOptions<PF extends ProofFormat> {
+export interface CreatePresentationOptions<PF extends ProofFormat> {
```

Comment:
CreatePresentationOptions was renamed to FormatCreatePresentationOptions to avoid clashes when exporting it (ProofService also has a `CreatePresentationOptions`.

I think in this file both are duplicated.